packages/tables/frontend/sirius-components-tables/src/rows/RowHeader.tsx

```diff
@@ -23,18 +23,15 @@ import { RowHeaderProps } from './RowHeader.types';
 export const RowHeader = ({ row, isExpanded, onClick, enableSubRows }: RowHeaderProps) => {
   const theme = useTheme();
   return (
-    <Box
-      display="flex"
-      alignItems="center"
-      justifyContent="space-between"
-      gap={theme.spacing(2)}
-      data-testid="table-row-header">
+    <Box display="flex" sx={{ height: '100%' }} alignItems="center" data-testid="table-row-header">
```

Comment:
It's a bit odd that this component needs to define how it should be laid out in its parent component